practices/development.md

```diff
@@ -132,48 +118,48 @@ Tasks to complete
   All deliverables should have either a set of concrete steps to take to meet the deliverable, or at least one task with the **acceptance criteria** for when the deliverable will be complete.
   Task lists should be in the top comment of the deliverable, and are encoded as markdown tasks lists (e.g. with `- [ ]`).
   Task lists should be updated over time as we learn the steps needed to close the deliverable.
-  For more complex deliverables, these tasks may be what goes onto the Sprint Board, rather than the deliverable itself.
+  For more complex deliverables, these tasks may be what goes onto the Cycle Board, rather than the deliverable itself.
 
 (coordination:deliverables-backlog)=
 ## The Team Backlog
 
 [Click here to go to the Team Backlog](https://github.com/orgs/2i2c-org/projects/22).
 
 The Team Backlog is a GitHub Projects Board with a list of [Deliverables and tasks](coordination:deliverables) across all of our active projects.
-This represents the work that the team is planning to do in the near future.
+This represents the work that the team is doing AND planning to do in the near future.
 These items adhere to the following principles:
 
 - The order of items should be roughly according to priority, with higher priority items at the top of lists.
 - Items on the board should have a **status** that signals whether they are ready to work or need more refinement before working.
-- If an item has multiple components or would otherwise take longer than a sprint to complete, create new issues as sub-tasks, and add *them* to the Sprint Board.
+- If an item has multiple components or would otherwise take longer than a cycle to complete, create new issues as sub-tasks, and add *them* to a dedicated [Project Backlog](coordination:project-backlog).
```

Comment:
> I think we should also state who is responsible for creating and updating this Project Backlog

I presume the main assignee since they have most of the context.

> Is this to be used in the planning meetings it's just for an easier coordination and keeping track of sub-tasks?

Easier coordination and keeping track... the parent issue (referencing the dedicated project board) should be the only one present in the team backlog.

> Also, what happens with these once they're done? Can we archive them, do we let them be, or just delete?

We should archive them because there is information there that might be useful for reporting purposes (among other things).

Reply:
Thanks for the extra context Damian. I think these dedicated boards are a great idea, but to be honest, I never created one myself. But I did use and update one created by Chris for the Toronto migration and I thought it was useful. I now feel more responsible and empowered doing it whenever I feel I need one. Thanks for this!